recipe/conda_build_config.yaml

```diff
@@ -65,12 +62,12 @@ cuda_compiler:
   - cuda-nvcc                  # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
We can also drop `nvcc` from `cuda_compiler` on the line above this

In CUDA 12, it was renamed to `cuda-nvcc` and has stayed the same since